src/main/java/com/royalrangers/bean/achievement/UserRewardBean.java

```diff
@@ -0,0 +1,17 @@
+package com.royalrangers.bean.achievement;
+
+import com.royalrangers.model.achievement.Reward;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class UserRewardBean {
+    private Long id;
+    private Date createDate;
+    private Date updateDate;
+    private UserAchievementBean user;
+    private Reward reward;
```

Comment:
@viachm we have talked about it previously. Entities are internal backend components. If we will do some work before saving, yes it's critical, and I think that we will do it. @chernienkoAndriy also, check other achievement beans and please avoid of using entities in beans. If this bean will contain one or two fields no need to create an additional one. In such case, you can write one base bean which will contain all fields which you need.